enginetest/script_queries.go

```diff
@@ -1665,4 +1665,25 @@ var CreateCheckConstraintsScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "duplicate indexes still returns correct results",
+		SetUpScript: []string{
+			"CREATE TABLE test (i int)",
+			"CREATE INDEX test_idx1 on test (i)",
+			"CREATE INDEX test_idx2 on test (i)",
+			"INSERT INTO test values (1), (2), (3)",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "SELECT * FROM test ORDER BY i",
+				Expected: []sql.Row{{1}, {2}, {3}},
+			},
+			{
+				Query: "EXPLAIN SELECT * FROM test where i = 2",
```

Comment:
Need a normal SELECT here, not an explain

Point is to make sure the query returns the right results